### PR TITLE
initial draft of influx.rb

### PR DESCRIPTION
**Pull request by randomprogrammer10**

### CODE REVIEW

File: back-end/influxdb.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'influxdb'
+
+class Influxdb
+  RASPBERRY_PI_IP = '172.168.38.35'.freeze
```
Comment (jlivermont) [inline back-end/influxdb.rb]: I don't think this is needed.

File: back-end/influxdb.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require 'influxdb'
+
+class Influxdb
```
Comment (jlivermont) [inline back-end/influxdb.rb]: Can we call this class something like `InfluxClient` or `InfluxAdapter`?

File: back-end/influxdb.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  RASPBERRY_PI_IP = '172.168.38.35'.freeze
+  HOST = 'localhost'.freeze
+  PORT = '8086'.freeze
+  RETRY_COUNT = '3'.freeze
+  TIME_OUT = '10'.freeze
+  DB_NAME = 'silverviper'.freeze
+
+  url = "https://#{RASPBERRY_PI_IP}/#{HOST}:#{PORT}/#{DB_NAME}?retry=#{RETRY_COUNT}"
```
Comment (jlivermont) [inline back-end/influxdb.rb]: Can we move this code into a constructor?

**Review by jlivermont — CHANGES_REQUESTED** [review]

File: back-end/influxdb.rb (hunk 2 above)
Comment (randomprogrammer10) [reply back-end/influxdb.rb]: > Can we call this class something like `InfluxClient` or `InfluxAdapter`?

Renamed to InfluxClient

File: back-end/influxdb.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+require 'influxdb'
```
Comment (jlivermont) [inline back-end/influxdb.rb]: There's no need for this class, this can go.

**Review by jlivermont — CHANGES_REQUESTED** [review]

File: back-end/influx_client.rb [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  DB_NAME = 'silverviper'.freeze
+  TABLE_NAME = 'weather_metrics'.freeze
+
+  def initialize
+    @url = "https://#{HOST}:#{PORT}/#{DB_NAME}?retry=#{RETRY_COUNT}"
+    @influxdb = InfluxDB::Client.new url: url, open_timeout: TIME_OUT
+  end
+
+  def get_weather_metric_data(metrics, start, stop)
+    select_query = generate_query_select_string(metrics.join(','))
```
Comment (jlivermont) [inline back-end/influx_client.rb:18]: What is this `generate_query_select_string` method?

**Review by jlivermont — APPROVED** [review]

**Pull request merged**